### PR TITLE
Skip passed test cases for secret scanner

### PR DESCRIPTION
**Pull request by harshilswan**

### CODE REVIEW

File: docs/docs/help/best_practices.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
+
+# Best Practices For Creating Rego Rules
```
Comment (farchide) [inline docs/docs/help/best_practices.md]: `Best Practices For Creating Rego Rules for Prancer Security Platform`

File: docs/docs/help/best_practices.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+
+Example
+
+- `PR-AWS-CFR-S3-001`
+
+## 2. Initialize default evalution variable
+
+- `default s3_accesslog = null`
+
+## 3. Write the rule to verify the your resources configuration
```
Comment (farchide) [inline docs/docs/help/best_practices.md]: `Write the rule to verify your resource configurations`

Comment (farchide) [inline docs/docs/help/best_practices.md]: mention here that we are always checking resource type

File: docs/docs/help/best_practices.md [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+}
+
+s3_accesslog_metadata := {
+    "Policy Code": "PR-AWS-CFR-S3-001",
+    "Type": "IaC",
+    "Product": "AWS",
+    "Language": "AWS Cloud formation",
+    "Policy Title": "AWS Access logging not enabled on S3 buckets",
+    "Policy Description": "Checks for S3 buckets without access logging turned on. Access logging allows customers to view complete audit trail on sensitive workloads such as S3 buckets",
+    "Resource Type": "",
```
Comment (farchide) [inline docs/docs/help/best_practices.md]: fill out the resource type

**Review by farchide — COMMENTED** [review]
@harshilswan please fix these issues and then move this file to `https://github.com/prancer-io/prancer-compliance-test/` repo under docs/help

**Review by ajeybk — APPROVED** [review]
Ok looks fine

**Review by vatsalgit5118 — APPROVED** [review]

**Pull request merged**